### ISSUE TITLE
[Bug] OpenMetadata fails to start on v1.5.0

### ISSUE THREAD

**Issue by cberge908** [open-metadata/OpenMetadata#17593]
**Affected module**
Backend & UI

**Describe the bug**
After updating our K8s deployment to use the most recent helm chart on v1.5.0 the openmetadata pod does not start anymore. The openmetadata-dependency pods all started well after updating them.

**To Reproduce**
1. Take an existing OpenMetdata deployment on K8s (e.g. v1.4.8)
2. Update the openmetadata-dependencies chart to v1.5.0
3. Update the openmetadata chart to v1.5.0
4. OpenMetadata pod fails to start

**Expected behavior**
Pod should start properly.

**Version:**
 - OS: K8s v1.30.3
 - Python version:  -
 - OpenMetadata version: v1.5.0
 - OpenMetadata Ingestion package version: v1.5.0

**Additional context**
The container does not seem to be able to bind port 8585 properly. This is indicated by the log files:

```txt
INFO [2024-08-27 12:27:10,157] [main] i.d.s.ServerFactory - Starting OpenMetadataApplication
ERROR [2024-08-27 12:27:10,235] [main] i.d.c.ServerCommand - Unable to start server, shutting down
java.lang.RuntimeException: java.io.IOException: Failed to bind to openmetadata/10.0.98.140:8585
	at org.eclipse.jetty.setuid.SetUIDListener.lifeCycleStarting(SetUIDListener.java:229)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.setStarting(AbstractLifeCycle.java:205)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at io.dropwizard.cli.ServerCommand.run(ServerCommand.java:52)
	at io.dropwizard.cli.EnvironmentCommand.run(EnvironmentCommand.java:68)
	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:98)
	at io.dropwizard.cli.Cli.run(Cli.java:78)
	at io.dropwizard.Application.run(Application.java:94)
	at org.openmetadata.service.OpenMetadataApplication.main(OpenMetadataApplication.java:616)
Caused by: java.io.IOException: Failed to bind to openmetadata/10.0.98.140:8585
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310)
	at org.eclipse.jetty.setuid.SetUIDListener.lifeCycleStarting(SetUIDListener.java:216)
	... 8 common frames omitted
Caused by: java.net.BindException: Address not available
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:555)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(ServerSocketChannelImpl.java:337)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:294)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:89)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344)
	... 10 common frames omitted
java.lang.RuntimeException: java.io.IOException: Failed to bind to openmetadata/10.0.98.140:8585
	at org.eclipse.jetty.setuid.SetUIDListener.lifeCycleStarting(SetUIDListener.java:229)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.setStarting(AbstractLifeCycle.java:205)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at io.dropwizard.cli.ServerCommand.run(ServerCommand.java:52)
	at io.dropwizard.cli.EnvironmentCommand.run(EnvironmentCommand.java:68)
	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:98)
	at io.dropwizard.cli.Cli.run(Cli.java:78)
	at io.dropwizard.Application.run(Application.java:94)
	at org.openmetadata.service.OpenMetadataApplication.main(OpenMetadataApplication.java:616)
Caused by: java.io.IOException: Failed to bind to openmetadata/10.0.98.140:8585
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310)
	at org.eclipse.jetty.setuid.SetUIDListener.lifeCycleStarting(SetUIDListener.java:216)
	... 8 more
Caused by: java.net.BindException: Address not available
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:555)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(ServerSocketChannelImpl.java:337)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:294)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:89)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344)
	... 10 more
INFO [2024-08-27 12:27:10,239] [JettyShutdownThread] i.d.h.c.HealthCheckManager - delayed shutdown: started (waiting 1 second)
INFO [2024-08-27 12:27:11,240] [JettyShutdownThread] i.d.h.c.HealthCheckManager - delayed shutdown: finished
```

**Comment by nicor88** [comment]
@cberge908 this seems related to https://github.com/open-metadata/openmetadata-helm-charts/issues/256, and there is a quick fix, proposed here https://github.com/open-metadata/openmetadata-helm-charts/pull/255

**Comment by tutte** [comment]
Fixed in 1.5.1
Thanks for reporting @nicor88 :pray:
